_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,167 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements follow content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility tree
+  - CSS Styling
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `ul` , `ol` , and `dl` [HTML elements][] for which
+
+- the [semantic role][] is the same as the [implicit semantic role][]; and
+- is [included in the accessibility tree][]; and
+- is [visible][].
```

Comment:
I'm not so sure about this. Wouldn't it make more sense for this to be applicable to all lists except those that have display:none or visibility:hidden?